Sources/ClientRuntime/Middleware/RetryerMiddleware.swift

```diff
@@ -0,0 +1,88 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+public struct RetryerMiddleware<Output: HttpResponseBinding,
```

Comment:
It'd be great to add docs to the public methods but i understand this is just copy paste from before.